cmd/validate/image_test.go

```diff
@@ -271,6 +272,44 @@ func Test_ValidateImageCommand(t *testing.T) {
 	  }`, mockPublicKey, mockPublicKey), out.String())
 }
 
+func Test_ValidateImageCommandKeyless(t *testing.T) {
```

Comment:
This test is timing out because the call to `NewPolicy` is triggering resolving certs and keys.